src/Microsoft.Azure.Relay/Common/InputQueue.cs

```diff
@@ -343,11 +336,11 @@ static void CompleteWaitersLater(bool itemAvailable, IQueueWaiter[] waiters)
         {
             if (itemAvailable)
             {
-                ActionItem.Schedule(s => CompleteWaitersTrueCallback(s), waiters);
+                ActionItem.Schedule(CompleteWaitersTrueCallback, waiters);
```

Comment:
Undo this change for fewer allocations as described above.

Reply:
Sure